### PR TITLE
fix material use

### PR DESCRIPTION
**Pull request by mirsfang**
This line of code appears to affect Android phones on Qualcomm's platform，This line of code will not work without it

### CODE REVIEW

**Comment by mirsfang** [comment]
My device is Samsung S20+(865) and Pixel 3xl (845) , build main branch android is not work , but  xiaomi k30 ultra is normal,  i find main branch this commit `57df4481e99dd45af04e47c9259b44187862bff5` results in，when i revert this line , every one is ok

File: filament/src/PostProcessManager.cpp [hunk 1]
```diff
@@ -1482,6 +1482,7 @@ void PostProcessManager::colorGradingSubpass(DriverApi& driver,  bool translucen
     Handle<HwRenderPrimitive> const& fullScreenRenderPrimitive = engine.getFullScreenRenderPrimitive();
 
     auto const& material = getPostProcessMaterial("colorGradingAsSubpass");
+    material.getMaterialInstance()->use(driver);
```
Comment (pixelflinger) [inline filament/src/PostProcessManager.cpp:1485]:
```suggestion
    FMaterialInstance* mi = material.getMaterialInstance();
    mi->use(driver);
```

**Review by romainguy — APPROVED** [review]

**Comment by pixelflinger** [comment]
nice fix thank you

**Comment by romainguy** [comment]
Thank you!

**Pull request merged**